From dd2cdb21df43bcb7544fa622abe5c3795e54eb1e Mon Sep 17 00:00:00 2001
From: IlyaSergeyev <ilya.sergeev@intel.com>
Date: Mon, 7 Nov 2022 12:32:46 +0800
Subject: [PATCH 1/2] FROMGIT: iHD: [VP9][Encode] Unaligned height corruption -
 turn on w/a for linux

We have different w/a turning on/off mechanism for Linux

(cherry picked from commit 0bf792a06680c5ac93dc5366c68638b5e5d80308 https://github.com/intel/media-driver.git)

BUG=b:231639580
TEST= emerge- libva-intel-media-driver
---
 media_driver/linux/gen12/ddi/media_sku_wa_g12.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/media_driver/linux/gen12/ddi/media_sku_wa_g12.cpp b/media_driver/linux/gen12/ddi/media_sku_wa_g12.cpp
index 8197400b4518..33b0b4ac1d66 100644
--- a/media_driver/linux/gen12/ddi/media_sku_wa_g12.cpp
+++ b/media_driver/linux/gen12/ddi/media_sku_wa_g12.cpp
@@ -361,6 +361,7 @@ static bool InitTglMediaWa(struct GfxDeviceInfo *devInfo,
     /*software wa to use huc copy for init aux table for MFX*/
     MEDIA_WR_WA(waTable, Wa_22010493002, 1);
 
+    MEDIA_WR_WA(waTable, Wa_Vp9UnalignedHeight, 1);
     return true;
 }
 
-- 
2.17.1

